peers-lib/src/main/java/net/sourceforge/peers/rtp/RtpSession.java

```diff
@@ -234,15 +234,15 @@ public Integer run() {
             int length = datagramPacket.getLength();
             byte[] trimmedData = new byte[length];
             System.arraycopy(data, offset, trimmedData, 0, length);
+            RtpPacket rtpPacket = rtpParser.decode(trimmedData);
             if (mediaDebug) {
                 try {
-                    rtpSessionInput.write(trimmedData);
+                    rtpSessionInput.write(rtpPacket.getData());
```

Comment:
I used to save the full rtp packet to have as much debug data as possible for debug, here you save only the rtp payload but you kept the _rtp_session.input suffix. I think it could be confusing. If you don't mind, I think we should keep the full rtp packet in this debug file.